### PR TITLE
chore: prepare release

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

We are releasing a new version to align with our decision to set the agent-js version as `^2.0.0`, allowing developers to choose any version in the `2.x.z` range.

# Changes

- Bumped versions in package.json
- Prepared an overview in the CHANGELOG
- Fixed a typo in the previous overview related to the utils version number

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,21 @@
-# Next
+# 2024.10.09-1100Z
+
+## Overview
+
+The current status of the libraries at the time of the release is as follows:
```
Comment (dskloetd) [inline CHANGELOG.md:5]: If the agent-js version is the reason for the release, should it be mentioned in the change log?

**Review by dskloetd — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG.md:5]: It's already mentionned:

> ## Build
>
> - Downgrade Agent-js to `^2.0.0` to let consumers pick the compatible version they wish to use.

Or do you mean something else?

Comment (dskloetd) [reply CHANGELOG.md:5]: Sorry. I was confused because GitHub had collapsed it.

**Comment by peterpeterparker** [comment]
I updated the release time to `1140Z` before merge.

**Pull request merged**